clipboard-apis/readText.tentative.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Clipboard API: navigator.clipboard.readText()</title>
+<meta name="timeout" content="long">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<body>
+  <h1>Test for navigator.clipboard.readText()</h1>
+
+  <script>
+    promise_test(async t => {
+      try {
+        const text = await navigator.clipboard.readText();
+        assert_equals(typeof text, "string", "readText() should resolve to a string");
+      } catch (e) {
+        assert_unreached("Clipboard readText() threw an error: " + e.message);
+      }
```

Comment:
isn't that already covered by 

https://github.com/web-platform-tests/wpt/blob/9195b5520e60443329d238dc6e9927038b7a9a6f/clipboard-apis/async-navigator-clipboard-basics.https.html#L169-L174